release/main.go

```diff
@@ -16,9 +16,9 @@ func main() {
 	fmt.Println("Vocab size:", tk.VocabSize())
 	// Vocab size: 30522
 	fmt.Println(tk.Encode("brown fox jumps over the lazy dog", false))
-	// [2829 4419 14523 2058 1996 13971 3899] [brown fox jumps over the lazy dog]
+	// &{[2829 4419 14523 2058 1996 13971 3899] [] [] [] [brown fox jumps over the lazy dog] []}
```

Comment:
what are the empty slices?

Reply:
Here: https://github.com/sunhailin-Leo/tokenizers/blob/main/tokenizer.go#L21